cirq-core/cirq/contrib/acquaintance/executor.py

```diff
@@ -100,6 +117,67 @@ def optimization_at(
         )
 
 
+class StrategyExecutorTransformer:
```

Comment:
Decorate the class with `transformers.transformer` decorator.
```suggestion
@transformers.transformer
class StrategyExecutorTransformer:
```